environments/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM ubuntu:latest
+FROM python:3.10
 
 #Install requirements
-RUN apt update && apt install -y git python3 python3-pip s3fs
+RUN apt update && apt install -y git
 
-#Install gnomonicus from git
+#Install gnomonicus from git (for now)
```

Comment:
Is this still the case? Looks commented out? Or is this used in devlopment

Reply:
The git install is used in development and is commented out below - it uses pypi